zipline/data/minute_bars.py

```diff
@@ -891,6 +891,14 @@ def __init__(self, rootdir, sid_cache_size=1000):
         self._last_get_value_dt_position = None
         self._last_get_value_dt_value = None
 
+        # This is to avoid any bad data or other performance-killing situation
+        # where there a consecutive streak of 0 (no volume) starting at an
+        # asset's start date.
+        # if asset 1 started on 2015-01-03 but its first trade is 2015-01-06
+        # 10:31 AM US/Eastern, this dict would store {1: 1420558260000000000},
+        # which is the nanosecond epoch of that date.
+        self._known_zero_volume_dict = {}
```

Comment:
I would recommend storing these as the minute since epoch, since that is what the frame of reference of the other values you are comparing against is.
This would avoid an extra `/ NANOS_IN_MINUTE` call every time it is used.